src/main/java/org/spongepowered/common/util/gen/ArrayMutableBlockBuffer.java

```diff
@@ -66,16 +70,10 @@ public ArrayMutableBlockBuffer(BlockPalette palette, Vector3i start, Vector3i si
         this.data = new CharBackingData(blocks);
     }
 
-    public ArrayMutableBlockBuffer(BlockPalette palette, Vector3i start, Vector3i size, int[] blocks) {
-        super(start, size);
-        this.palette = palette;
-        this.data = new IntBackingData(blocks);
-    }
-
     ArrayMutableBlockBuffer(BlockPalette palette, Vector3i start, Vector3i size, BackingData blocks) {
         super(start, size);
         this.palette = palette;
-        this.data = blocks.copyOf();
```

Comment:
If you're going to change this then make it consistent with the immutable version (ie. remove the ctor in the immutable buffer that clones the data).